src/psyclone/tests/test_files/dynamo0p3/testkern_write_any_w2trace_mod.f90

```diff
@@ -47,14 +47,15 @@ module testkern_write_any_w2trace_mod
 
   type, extends(kernel_type) :: testkern_write_any_w2trace_type
      private
-     type(arg_type), dimension(7) :: meta_args = (/ &
+     type(arg_type), dimension(8) :: meta_args = (/ &
           arg_type(gh_field, gh_inc,  any_space_1), &
           arg_type(gh_field, gh_read, w0),          &
           arg_type(gh_field, gh_read, w1),          &
           arg_type(gh_field, gh_inc,  w2trace),     &
           arg_type(gh_field, gh_read, wtheta),      &
           arg_type(gh_field, gh_read, w2h),         &
-          arg_type(gh_field, gh_read, w2v)          &
+          arg_type(gh_field, gh_read, w2v),         &
```

Comment:
Is there a particular reason why you've modified this test kernel?

Reply:
It was practical to check the continuity of `W2Htrace` in `test_arg_discontinuous` (just parsing `1.5.4_single_invoke_write_anyspace_w2trace.f90` instead of adding a new example).

Reply:
OK, thanks.